README.md

```diff
@@ -160,6 +160,47 @@ describe('Example', () => {
 This means that the current screenshot is saved in the actual folder and you **manually need to copy it to your baseline**.
 If you instantiate `wdio-image-comparison-service` with `autoSaveBaseline: true` the image will automatically be saved into the baseline folder.
 
+### Running Programmatically 
+
+Here is a minimal example usage of getting `wdio-image-comparison-service` to work via `remote` options
+
+```js
+const {
+  remote
+} = require("webdriverio");
+
+const WdioImageComparisonService = require('wdio-image-comparison-service').default;
+
+let wdioImageComparisonService = new WdioImageComparisonService({});
+
+async function main() {
+  const browser = await remote({
+    logLevel: "silent",
+    capabilities: {
+      browserName: "chrome"
+    }
+  });
+
+  global.browser = browser;
+
+  wdioImageComparisonService.defaultOptions.autoSaveBaseline = true;
+  browser.defaultOptions = wdioImageComparisonService.defaultOptions;
+  browser.folders = wdioImageComparisonService.folders;
+  
+  wdioImageComparisonService.before(browser.capabilities)
+
+  await browser.url('https://webdriver.io/');
+  await browser.saveFullPageScreen('examplePaged', {});
+  await browser.checkFullPageScreen('examplePaged', {});
```

Comment:
Wow,

Thanks for adding this!

Can you maybe add the following comment?

```js
// or use this for ONLY saving a screenshot
 await browser.saveFullPageScreen('examplePaged', {});
// or use this for validating. Both methods don't need to be combined, see the FAQ
  await browser.checkFullPageScreen('examplePaged', {});
```

Reply:
👍 done